telegram/utils/helpers.py

```diff
@@ -55,6 +56,79 @@ def get_signal_name(signum: int) -> str:
     return _signames[signum]
 
 
+def is_local_file(obj: Optional[Union[str, Path]], absolute: bool = True) -> bool:
+    """
+    Checks if a given string is a file on local system.
+
+    Args:
+        obj (:obj:`str`): The string to check.
+        absolute (:obj:`bool`): Optional. Whether to allow only absolute paths. Defaults to
+            :obj:`True`.
+    """
+    if obj is None:
+        return False
+
+    path = Path(obj) if isinstance(obj, str) else obj
```

Comment:
I think bytes can also be a filename.

Maybe just:
```
path = Path(obj)
```

Reply:
jup, forgot that `Path(Path())` works …